docs/companion.md

```diff
@@ -457,6 +457,10 @@ The name of the bucket to store uploaded files in.
 
 The datacenter region where the target bucket is located.
 
+##### `s3.region` `COMPANION_AWS_PREFIX`
```

Comment:
Is this correct? `region` and `prefix` sound like very different things?

Reply:
whoops my bad